src/sass/base/_container.scss

```diff
@@ -40,5 +39,8 @@
       padding: 0 74px;
     }
   }
+  @media screen and (min-width: 1200px) {
+    min-width: 1200px;
```

Comment:
dodałem tutaj minimalną szerokość bo zaciągało mi wcześniej tą tabletową, zobaczcie czy to tak może być ;)